roles/db/tasks/mysql.yml

```diff
@@ -83,4 +83,15 @@
     name: 'all'
     target: /var/lib/jiskefet/sql/create_db_charset_utf8mb4.sql
   notify: restart mysql
+
+- name: Add daily backup cron job
+  cron:
+    name: "jiskefet-db-backup"
+    user: '{{ jiskefet_api_general_settings.TYPEORM_USERNAME }}'
```

Comment:
![image](https://user-images.githubusercontent.com/28677235/53454131-85d4e100-3a26-11e9-8680-344a1eb1ff58.png)

Reply:
Another possibility is to omit the user parameter and let it default to `root`, not sure what the complications are of this action yet.

Reply:
Does the jiskefet-api user have the permissions to read from the DB? If so, this would be nicer than root, and we can put the backup files in the user home dir

Reply:
The jiskefet-api user currently does not have permission to read from the database, minor changes are needed in the mysql.yml task.